apps/kol-amelamdim/pages/api/login.ts

```diff
@@ -21,18 +21,19 @@ export default async function handler(
         user.password
       );
       if (validPassword) {
-        const token = jwt.sign(
-          {
-            email: req.body.email,
-          },
-          process.env.ACCESS_TOKEN_SECRET,
-          {
-            expiresIn: '365',
-          }
-        );
+        const iat = Math.floor(Date.now() / 1000);
+        const exp = iat + 1440 * 365;
```

Comment:
I would stick to the 60 * 60 * 24 * 365 way because it׳s more understandable what every number are. Or instead I would set 1440 inside variable with self explanatory name